src/runtime_src/core/edge/drm/zocl/zocl_kds.c

```diff
@@ -582,14 +583,7 @@ static void notify_execbuf(struct kds_command *xcmd, int status)
 	struct kds_client *client = xcmd->client;
 	struct ert_packet *ecmd = (struct ert_packet *)xcmd->execbuf;
 
-	if (status == KDS_COMPLETED)
-		ecmd->state = ERT_CMD_STATE_COMPLETED;
-	else if (status == KDS_ERROR)
-		ecmd->state = ERT_CMD_STATE_ERROR;
-	else if (status == KDS_TIMEOUT)
-		ecmd->state = ERT_CMD_STATE_TIMEOUT;
-	else if (status == KDS_ABORT)
-		ecmd->state = ERT_CMD_STATE_ABORT;
+	ecmd->state = kds_ert_table[status];
```

Comment:
Please check the `status `variable to see if the index will fix in the array.  If not then bad things will happen (e.g., crash)